concepts/erlang-libraries/about.md

```diff
@@ -51,3 +52,5 @@ Elixir interoperability with Erlang libraries is not limited to Erlang's standar
 [erl-queue]: http://erlang.org/doc/man/queue.html
 [erl-crypto]: http://erlang.org/doc/man/crypto.html
 [erl-zip]: http://erlang.org/doc/man/zip.html
+[erl-calendar]: https://erlang.org/doc/man/calendar.html
+[erl-calendar-iso-week-number]: https://erlang.org/doc/man/calendar.html#iso_week_number-1
```

Comment:
After the addition of `Date.to_gregorian_days` and `DateTime.to_gregorian_seconds` in Elixir 1.11, I think `iso_week_number` is the only remaining Erlang calendar function that doesn't have an Elixir equivalent.